### PR TITLE
Add API endpoint to get list of employees

### PR DESCRIPTION
**Pull request by mhpjay422**
Why this PR?
- The Flatfile configuration needs this endpoint to validate existing employees against the current sheet

### CODE REVIEW

File: pages/api/v1/list-employees.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { prismaClient } from "../../../lib/prisma-client";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (typeof req.query.userId !== "string") {
```
Comment (johnmosesman) [inline pages/api/v1/list-employees.ts:8]: Why checking for string instead of checking for null or undefined?

**Review by johnmosesman — APPROVED** [review]
one comment, otherwise +1

File: pages/api/v1/list-employees.ts (hunk 1 above)
Comment (mhpjay422) [reply pages/api/v1/list-employees.ts:8]: @johnmosesman 

Theres a type coming off of NextApiRequest for query where string[] is a possibility. 

I had the conditional with a bunch of OR, but ended up using ☝️ 

CleanShot 2023-07-28 at 12 35 56

Comment (mhpjay422) [reply pages/api/v1/list-employees.ts:8]: Edited. Made it a bit more compact

**Pull request merged**